config/esp32/components/chip/Kconfig

```diff
@@ -320,14 +320,46 @@ menu "CHIP Device Layer"
                 Interval.
 
         config ICD_FAST_POLL_INTERVAL
-            int "ICD Fast Polling Interval"
+            int "ICD Fast Polling Interval(ms)"
             depends on ENABLE_ICD_SERVER
             default 200
             help
                 Fast Polling defines the fastest frequency at which the device can receive messages
                 in Active Mode. The Fast Polling interval SHALL be smaller than the Active Mode
                 Interval.
 
+        config ICD_IDLE_MODE_INTERVAL
+            int "ICD Idle Mode Interval(s)"
+            depends on ENABLE_ICD_SERVER
+            default 2
+            help
+                This value indicates the maximum interval in seconds the server can stay in idle
+                mode.
+
+        config ICD_ACTIVE_MODE_INTERVAL
```

Comment:
for new config, please absolutely add unit of measurement. these should end in _MS (here and below)

Reply:
Added